src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -189,28 +187,30 @@ namespace AppInstaller::CLI::Workflow
         {
             AICLI_LOG(CLI, Info, << "Installer hash verified");
             context.Reporter.Info() << Resource::String::InstallerHashVerified << std::endl;
+
+            WI_SetFlag(context.GetFlags(), Execution::ContextFlag::InstallerHashMatched);

```

Comment:
>WI_SetFlag [](start = 12, length = 10)

If this is suddenly going to become this prevalent, can we actually have GetFlags return by value, and have a SetFlag method?  I'm just imagining a desire to update said function with logging or some other behavior and that is so much harder when the assignments happen out here.